data-api/schemas/crm14.yml

```diff
@@ -470,8 +470,6 @@ Crm15DetailsDTO:
   properties:
     income1:
       $ref: "#/Crm15Income1DTO"
-    income2:
-      $ref: "#/Crm15Income2DTO"
     outgoings:
       $ref: "#/Crm15OutgoingsDTO"
```

Comment:
I don't think this has to be deleted. And should be out of the scope of this ticket. Please, get confirmation

Reply:
Fair point,  When i tried running locally, I found some issues, I spoke to @solomonmanik and put a small fix to resolve the issue, it related to EMP-321. I'm happy to remove and will raise a another PR, if required, but this issue needs to fixed ASAP too.

Reply:
@gpjustice Pls remove this change

Reply:
It is still not clear if that attribute needs to stay or not.
What tests were failing?